go/vt/sqlparser/keywords.go

```diff
@@ -592,6 +592,22 @@ var keywords = []keyword{
 	{"stored", STORED},
 	{"straight_join", STRAIGHT_JOIN},
 	{"stream", STREAM},
+	{"st_geomfromtext", ST_GeomFromText},
+	{"st_geomcollfromtext", ST_GeomCollFromText},
+	{"st_geometrycollectionfromtext", ST_GeometryCollectionFromText},
```

Comment:
You can roll these two together. They also do the same thing. Also the next one.
Just have one token for all the functions that are equivalent. This will allow you to have only one line for each of these in sql.y instead of repeating them there too.